.goreleaser.yaml

```diff
@@ -68,7 +72,7 @@ checksum:
   name_template: "checksums.txt"
 
 snapshot:
-  name_template: "{{ incpatch .Version }}-next"
+  version_template: "{{ incpatch .Version }}-next"
```

Comment:
side-track, this was deprecated in some Gorelease version. I don't think we're using snapshot releases, but this will squelch the warning.